frame/dex-router/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+//! # DEX Router Pallet
+//!
+//! Is used to add route to DEX for given asset_id's pair.
```

Comment:
please add reason why we have this pallet - to have permissions approved pools

Reply:
Is it permission approved pools or permission approved routes?

Reply:
permissioned approval of routes, pools are still freely creatable